manifests/07-operator.yaml

```diff
@@ -63,6 +63,25 @@ spec:
           requests:
             memory: "100Mi"
             cpu: "10m"
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /healthz
+            port: 8443
+            scheme: HTTPS
+          initialDelaySeconds: 150
```

Comment:
Why delay 150 seconds?

I think we can just take the defaults for the props other than `httpGet`

Reply:
Copy/paste from console.  Not sure why we had such a long initial delay there?

Reply:
updated